### DISCUSSION TITLE
Engaged status condition

### DISCUSSION THREAD

**Discussion by GregorHak** [Jagusti/fvtt-wfrp4e-gmtoolkit#246] [General]
Does Engaged is somehow taken into account by the foundry? What I mean is automatic counting, checking for talents, advantages, effects, spells, or does it serve mainly as a visual symbol? And if you forget to assign it automatically, you won't miss anything.
![obraz](https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/assets/99994770/e75fadeb-bbd6-452b-bb15-9f67a68c493a)

**Comment by GregorHak** [comment]
@Jagusti

**Reply by Jagusti** [reply]
Hey there. I'm pretty sure I don't use it for advantage automation. It may be used in other parts of the system or by some effects scripts. Couldn't tell you without checking the system repo, which I can't do at the moment. It's a system feature though, definitely, rather than a Toolkit addition.
